lib/jsdom/living/window/Location-impl.js

```diff
@@ -26,11 +25,11 @@ exports.implementation = class LocationImpl {
     return this._locationObjectNavigate(url);
   }
 
-  _locationObjectNavigate(url/* , { replacement = false } = {} */) {
+  _locationObjectNavigate(url, { replacement = false } = {}) {
     // Not implemented: the setup for calling navigate, which doesn't apply to our stub navigate anyway.
     // Not implemented: using the replacement flag.
```

Comment:
I was passing it through ready to use in `navigateFetch`, but since real browsers seem to use it for navigateToFragment as well, I've updated the implementation so it does actually use the `replace` flag.